plugins/modules/elb_target_group.py

```diff
@@ -741,12 +741,18 @@ def create_or_update_target_group(connection, module):
     if stickiness_enabled is not None:
         if stickiness_enabled and current_tg_attributes['stickiness_enabled'] != "true":
             update_attributes.append({'Key': 'stickiness.enabled', 'Value': 'true'})
-    if stickiness_lb_cookie_duration is not None:
-        if str(stickiness_lb_cookie_duration) != current_tg_attributes['stickiness_lb_cookie_duration_seconds']:
-            update_attributes.append({'Key': 'stickiness.lb_cookie.duration_seconds', 'Value': str(stickiness_lb_cookie_duration)})
-    if stickiness_type is not None and "stickiness_type" in current_tg_attributes:
-        if stickiness_type != current_tg_attributes['stickiness_type']:
-            update_attributes.append({'Key': 'stickiness.type', 'Value': stickiness_type})
+    else:
+        if current_tg_attributes.get('stickiness_enabled') == "true":
+            stickiness_enabled = True
```

Comment:
I'm not a fan of reassigning this var, this could easily cause confusion as it's been set to the module_params value up until this point. We could avoid this entire var assignment and the if on the next line by moving everything that's currently at ln750-755 into this `if`.